FHEM/90_SIGNALduino_un.pm

```diff
@@ -203,12 +203,14 @@ SIGNALduino_un_Parse($$)
 	my @delevopmentargs = split (",",$value);
 	my $search = "y".$protocol;			# for version 2
 	my $search2 = "u".$protocol;		# for version 1
+	my $search3 = "m".$protocol;		# for version 3
 
-	if ((any{ $search eq $_ }@delevopmentargs) || (any{ $search2 eq $_ }@delevopmentargs)) {
+	if ((any{ $search eq $_ }@delevopmentargs) || (any{ $search2 eq $_ }@delevopmentargs) || (any{ $search3 eq $_ }@delevopmentargs)) {
```

Comment:
So richtig gefällt mir das mittlerweile nicht mehr.

Wäre es ausreichend mit einer Regex zu prüfen, ob es die ID mit einem der Buchstaben gibt?

So ins balue
=~ m/([myp])$protocol/g

Dann würde in $1 der Bezeichner stehen. Die ID steht ja bereits in einer Variable.

https://regex101.com/r/cWrD7H/2